privacy/ccpa/ccpa.go

```diff
@@ -0,0 +1,34 @@
+package ccpa
+
+import (
+	"encoding/json"
+
+	"github.com/buger/jsonparser"
+	"github.com/mxmCherry/openrtb"
+)
+
+// Policy represents the CCPA regulation of an OpenRTB bid request.
+type Policy struct {
+	Enabled bool
+	Value   string
+}
+
+// Write mutates an OpenRTB bid request with the context of the CCPA policy.
+func (p Policy) Write(req *openrtb.BidRequest) error {
+	if p.Enabled == false {
+		return nil
+	}
+
+	if req.Regs == nil {
+		req.Regs = &openrtb.Regs{}
+	}
+
+	if req.Regs.Ext == nil {
+		req.Regs.Ext = json.RawMessage(`{"us_privacy":"` + p.Value + `"}`)
```

Comment:
It's a bit strange how the CCPA regulation maps to the `us_privacy` symbol. Should we call it USPrivacy in all of the models or keep the CCPA terminology?

Reply:
I think the idea is that several states are enacting similar laws, and we will just filter based on the most strict version of the law as "us_privacy" signal.

Reply:
Cool. Kept CCPA in code for now.